Gems/Atom/Feature/Common/Code/Include/Atom/Feature/SplashScreen/SplashScreenSettings.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/SystemAllocator.h>
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace AZ::Render
+{
+    class SplashScreenSettings
+    {
+    public:
+        AZ_RTTI(AZ::Render::SplashScreenSettings, "{4B441F73-8F72-4DCA-9451-416C40693487}");
+        AZ_CLASS_ALLOCATOR(AZ::Render::SplashScreenSettings, AZ::SystemAllocator, 0);
+
+        SplashScreenSettings() = default;
+        virtual ~SplashScreenSettings() = default;
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        AZStd::string m_imagePath;
+        float m_lastingTime = 10.0f;
```

Comment:
Assuming seconds. Does this behave as a minimum time? Some usefully of splash screen is something to look at while the game is loading. In that case, a splash screen may have a minimum time (so it doesn't flash for hal;f a second if loading is super fast), but generally it's useful to fade it out as soon as the loading has finished. For example, in LY1.28 you could set a global splashscreen that would automatically appear during app startup and level loads. Each level could also override the splashscreen using a level.cfg.

Reply:
+1 
```suggestion
        float m_secondsTillFadeOut= 10.0f;
```

Reply:
We need a minimal set of resources that can be loaded to render the splash screen. But now the engine reaches the MainLoop only after everything are initialized. We should need some further work to allow Atom to take some priority, something like preInit/postInit.